### ISSUE TITLE
The selected value is not announced when navigating to the pick model dropdown: A11y_VS Code_Copilot Chat_Screen Reader

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#237141]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM-VisualStudioCodeClient-Win32-Jan2024; #Visual Studio Code Client; #Win32; #Win11; #WCAG1.3.1; #FTP; #Element:Dropdown; #Info and Relationships; #Screen Reader; #NVDA; #ED:Bug9576428

### Environment and OS details:
Version: 1.97.0-insider (user setup)
Windows 11 Enterprise Version 24H2
OS Build: 26100.2605
Screen Reader: NVDA

### Repro Steps:

1. Turn on NVDA/JAWS.
2. Open Visual Studio Code Insiders editor.
3. Tab till Copilot chat and press enter.
4. In GitHub Copilot Chat, sign in with the GitHub account (Microsoft email account)
5. Navigate to Chat(Ctrl+Alt+I) in Left Nav and and activate it.
6. Navigate to Pick Model Button.
7. Observe that the selected value is not announced when navigating to the pick model dropdown.

### Actual Result:
The selected value is not announced when navigating to the pick model dropdown.

### Expected Result:
The selected value should be announced when navigating to the pick model dropdown.

### User Impact:
Screen reader users will face difficulty if the selected value is not announced when navigating to the pick model dropdown.

Attachment:
![Image](https://github.com/user-attachments/assets/a3f0d600-aaa8-4235-ba68-ac906e80c550)

**Comment by meganrogge** [comment]
In your gif, you do not open the menu, so nothing is read out. 

See below, this is working for me. I suggest closing the issue.

https://github.com/user-attachments/assets/b62a3959-1533-4728-a8cc-ce4a51d57ba6

**Comment by meganrogge** [comment]
Does not reproduce for me on windows either
